meson.build

```diff
@@ -201,7 +201,6 @@ subdir('init.d')
 subdir('local.d')
 subdir('man')
 subdir('pkgconfig')
-subdir('scripts')
```

Comment:
same comment as the compiled code ... having a hard time seeing this as improvement and not just a lot of busywork

Reply:
oo